### PR TITLE
✨Allow amp-selector in AMP4ADS

### LINKED ISSUE #25773: Allow amp-selector in AMP4ADS
**amp-form** is allowed in amp4ads, and the doc says amp-selector is allowed in amp-form:
https://amp.dev/documentation/components/amp-form/?format=ads#inputs-and-fields

Is there any reason why **amp-selector** itself is not allowed in amp4ads?

### PR DESCRIPTION
**Pull request by jyn15**
Resolves #25773: updates validator spec and tests to allow amp-selector in AMP4ADS.

### CODE REVIEW

File: validator/testdata/amp4ads_feature_tests/amp_selector.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```
Comment (honeybadgerdontcare) [inline validator/testdata/amp4ads_feature_tests/amp_selector.html]: please add the license file here

Comment (jyn15) [reply validator/testdata/amp4ads_feature_tests/amp_selector.html]: Done. Also updated the list in extensions.html.

Comment (honeybadgerdontcare) [reply validator/testdata/amp4ads_feature_tests/amp_selector.html]: Forgot about this test for allowed extensions in AMP Ads: https://github.com/ampproject/amphtml/blob/master/validator/engine/validator_test.js#L1265

Comment (jyn15) [reply validator/testdata/amp4ads_feature_tests/amp_selector.html]: Done. Thanks!

File: validator/testdata/amp4ads_feature_tests/amp_selector.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
Comment (honeybadgerdontcare) [inline validator/testdata/amp4ads_feature_tests/amp_selector.html]: nit: update to 2019. don't need to run update-tests again, just update in the .out file too

Comment (jyn15) [reply validator/testdata/amp4ads_feature_tests/amp_selector.html]: Done. Thanks!

**Pull request merged**